chcemvediet/apps/inforequests/admin.py

```diff
@@ -499,35 +445,22 @@ def delete_selected(self, request, queryset):
         for obj in queryset:
```

Comment:
@viliambalaz Riadok
```
@decorate(short_description=u'Delete selected actions')
```
mozeme IMHO aj pre tento admin vyhodit a nechat defaultny nazov akcie "Delete selected". Tak ako to je teraz vo vsetkych ostatnych adminoch pri `delete_selected`

Reply:
:ok: